gym/train.sh

```diff
@@ -0,0 +1,3 @@
+chmod 777 -Rf tf_od_api/workspace/train_core/
+nvidia-docker run -it -v $(pwd)/tf_od_api/workspace/train_core:/home registry.fsoc.hpi.uni-potsdam.de/coral-ai > tf_od_api/workspace/train_core/train.log
```

Comment:
Just like in `eval.sh`, you probably want to have `$(pwd)` in the output directory as well

Reply:
the redirection  has been removed, we only used $(pwd) because docker require it when mounting and binding a directory